packages/graphql_server/src/LinkedInScrapingService.ts

```diff
@@ -0,0 +1,73 @@
+import axios, { AxiosRequestConfig } from 'axios'
+import { LinkedInCompanyProfile, CompanyDataResponse } from '../types/linkedInScraping'
+
+const username = process.env.SCRAPING_BOT_USER_NAME || ''
+const apiKey = process.env.SCRAPING_BOT_API_KEY || ''
+const apiEndPoint = process.env.SCRAPING_BOT_API_ENDPOINT || ''
+const auth = 'Basic ' + Buffer.from(username + ':' + apiKey).toString('base64')
+
+function sleep(ms: number): Promise<void> {
+  return new Promise((resolve) => {
+    setTimeout(resolve, ms)
+  })
+}
+
+/**
+ * There is an inital POST with the scraping parameters to get a unique responseId from the API.
+ * The second GET request then finally returns the scraping result.
+ * It is important to note, that the API is using an Array of objects for the data attribute.
+ * According to their documentation, they needed to implement it in this two-request-way, because LinkedIn uses protection tools to avoid bots
+ * https://www.scraping-bot.io/web-scraping-documentation/#data-scraper-api
+ * @param linkedInHandle
+ * @returns the specified LinkedInCompanyProfileType including the most important data
+ */
+export async function getCompanyInfosFromLinkedIn(
+  linkedInHandle: string
+): Promise<LinkedInCompanyProfile> {
```

Comment:
I don't understand why it is returning a promise... @pczern @jonassuess does one of you know?